pkg/cli/admin/upgrade/status/examples/4.16.0-ec2-control-plane-updated-pdb-prohibits-draining.detailed-output

```diff
@@ -0,0 +1,100 @@
+build0-gstfj-ci-tests-worker-d-r2k42                              Outdated      Pending    4.16.0-ec.2   ?      
+build0-gstfj-ci-tests-worker-d-r9zqc                              Outdated      Pending    4.16.0-ec.2   ?      
+build0-gstfj-ci-tests-worker-d-w2bgd                              Outdated      Pending    4.16.0-ec.2   ?      
+build0-gstfj-w-d-7rj9w.c.openshift-ci-build-farm.internal         Outdated      Pending    4.16.0-ec.2   ?      
+build0-gstfj-ci-builds-worker-b-65df4                             Completed     Updated    4.16.0-ec.3   -      
+build0-gstfj-ci-builds-worker-b-dvrd2                             Completed     Updated    4.16.0-ec.3   -      
+build0-gstfj-ci-builds-worker-b-f7rlh                             Completed     Updated    4.16.0-ec.3   -      
+build0-gstfj-ci-builds-worker-b-m4vgr                             Completed     Updated    4.16.0-ec.3   -      
+build0-gstfj-ci-builds-worker-b-mg8bd                             Completed     Updated    4.16.0-ec.3   -      
+build0-gstfj-ci-builds-worker-c-2hzgm                             Completed     Updated    4.16.0-ec.3   -      
+build0-gstfj-ci-builds-worker-c-dmjft                             Completed     Updated    4.16.0-ec.3   -      
+build0-gstfj-ci-builds-worker-c-ng8s5                             Completed     Updated    4.16.0-ec.3   -      
+build0-gstfj-ci-builds-worker-c-vmmfj                             Completed     Updated    4.16.0-ec.3   -      
+build0-gstfj-ci-builds-worker-d-4pjlz                             Completed     Updated    4.16.0-ec.3   -      
+build0-gstfj-ci-builds-worker-d-z49wm                             Completed     Updated    4.16.0-ec.3   -      
+build0-gstfj-ci-tests-worker-b-d9vz2                              Completed     Updated    4.16.0-ec.3   -      
+build0-gstfj-ci-tests-worker-b-fkn28                              Completed     Updated    4.16.0-ec.3   -      
+build0-gstfj-ci-tests-worker-b-mc7fm                              Completed     Updated    4.16.0-ec.3   -      
+build0-gstfj-ci-tests-worker-b-pqz8t                              Completed     Updated    4.16.0-ec.3   -      
+build0-gstfj-w-b-f7dkq.c.openshift-ci-build-farm.internal         Completed     Updated    4.16.0-ec.3   -      
+build0-gstfj-w-b-infra-2stfh.c.openshift-ci-build-farm.internal   Completed     Updated    4.16.0-ec.3   -      
+build0-gstfj-w-b-infra-mhb8g.c.openshift-ci-build-farm.internal   Completed     Updated    4.16.0-ec.3   -      
+build0-gstfj-w-b-infra-n9kc5.c.openshift-ci-build-farm.internal   Completed     Updated    4.16.0-ec.3   -      
+build0-gstfj-w-b-mwf8t.c.openshift-ci-build-farm.internal         Completed     Updated    4.16.0-ec.3   -      
+build0-gstfj-w-c-nsmn5.c.openshift-ci-build-farm.internal         Completed     Updated    4.16.0-ec.3   -      
+build0-gstfj-w-c-qjbdl.c.openshift-ci-build-farm.internal         Completed     Updated    4.16.0-ec.3   -      
```

Comment:
This is the only output that differs from its non-detailed variant (for now)

Reply:
This is good. When we have a detailed output how do we tell the user where to focus or what is the main toot cause of these issues. Because I am familier with OCP I can guess that the issue is 

```
 Reason: ClusterOperatorDegraded
  Message: Cluster operator machine-config is degraded
```

But will that be obvious to the user as well?

Reply:
Hopefully the remaining bits from https://github.com/openshift/oc/pull/1690 (`--details=health` or `--details=all`) will help with that, expanding the `= Update Health =` section items with "description", "reference" and "resources" fields. That PR grew too much so I'm breaking it down to smaller, easier ones. This PR just brings the change to implement turning off the omission:

```
Omitted additional 49 Total, 22 Completed, 46 Available, 3 Progressing, 27 Outdated, 3 Draining, 0 Excluded, and 0 Degraded nodes.
Pass along --details=nodes to see all information.
``` 

I'm convinced that "I want to see all nodes" and "I want to see details about health insights" are separate use cases so I split them this way.

Reply:
Check out where #1690 eventually takes this:

https://github.com/openshift/oc/blob/6da69b9f47c335062a93d7e1e842f8abc99bcd6b/pkg/cli/admin/upgrade/status/examples/4.16.0-ec2-control-plane-updated-pdb-prohibits-draining.detailed-output#L92-L99

Reply:
@petr-muller sounds good to me.